itest/lnd_onchain_test.go

```diff
@@ -278,9 +279,19 @@ func runCPFP(ht *lntest.HarnessTest, alice, bob *node.HarnessNode) {
 		Outpoint: op,
 		// We use a higher fee rate than the default max and expect the
 		// sweeper to cap the fee rate at the max value.
-		SatPerVbyte: maxFeeRate * 2,
+		SatPerVbyte:  maxFeeRate * 2,
+		IncludeRawTx: true,
 	}
-	bob.RPC.BumpFee(bumpFeeReq)
+	bumpFeeResp := bob.RPC.BumpFee(bumpFeeReq)
+
+	// The sweep tx hex on success should be set and be valid.
+	require.NotEmpty(ht, bumpFeeResp.SweepTxHex)
+	rawTxBytes, err := hex.DecodeString(bumpFeeResp.SweepTxHex)
+	require.NoError(ht, err, "bumpfee sweepTxHex invalid hex")
+	rawTx := &wire.MsgTx{}
+	err = rawTx.Deserialize(bytes.NewReader(rawTxBytes))
+	require.NoError(ht, err, "bumpfee sweepTxHex invalid")
+	require.Equal(ht, bumpFeeResp.SweepTxHex, rawTx.TxHash().String())
```

Comment:
the reason this itest doesnt pass is cause it now waits for the sweep tx to be created which as you mentioned in the docs takes at least 30 seconds. But the itest `BumpFee` call times out after 30 seconds.

Reply:
nit: also, pls can you space things out a bit more for readability?

```
        // The sweep tx hex on success should be set and be valid.
	require.NotEmpty(ht, bumpFeeResp.SweepTxHex)

	rawTxBytes, err := hex.DecodeString(bumpFeeResp.SweepTxHex)
	require.NoError(ht, err, "bumpfee sweepTxHex invalid hex")

	rawTx := &wire.MsgTx{}
	err = rawTx.Deserialize(bytes.NewReader(rawTxBytes))
	require.NoError(ht, err, "bumpfee sweepTxHex invalid")
	require.Equal(ht, bumpFeeResp.SweepTxHex, rawTx.TxHash().String())
```

Reply:
Thanks! Also Gonna space things out in [`itest/lnd_channel_force_close_test`](https://github.com/lightningnetwork/lnd/blob/5cb4811e86ed1607c1c9671756d7276256e2ec44/itest/lnd_channel_force_close_test.go#L149-L162) since it is related